Pontoon update: Moved resources content into ui (#226)
mozilla-l10n/localizer-documentation: src/tools/pontoon/ui.md

* Clicking the keyboard icon displays a list of available shortcuts.
* The numbers displayed to the right of the keyboard icon (e.g. `50|59`) are the number of characters in the target and source string.

### Translation list

The space below the editor displays the list of suggestions or translations for the current string.

Each entry contains:
* The name of the translator and their profile picture (linking to their profile).
* How long ago the entry was submitted (hover over to see the full date and time as a tooltip).
* The translation.
* Icons indicating translation status (see below).
* [Translation comments](glossary.md#comment).

Icons to the right indicate the status of each translation:
* The solid green circle with checkmark indicates that the translation has been approved.
* The outlined lime green circle with checkmark indicates a pretranslation that has not yet been reviewed.
* If both icons are gray, translation has been suggested but not yet reviewed.
* The red cross indicates that the translation has been rejected. The entire element will look opaque.

> * The trashcan, available only for rejected translations, can be used to completely delete a translation.

Author: Added `Those with the contributor role can only remove their own translations, while those with a translator, team manager, or administrator role can delete anyone's.`

## Translation tools and comments

Built-in translation tools are located in the rightmost column. For more information about `MACHINERY`, and `LOCALES`, see the page dedicated to [Pontoon’s translation resources](resources.md).
Built-in translation tools are located in the rightmost column.

> ### Terminogy

Author: 😏

The `COMMENTS` tab is used to display existing **source string comments**, or add new ones. Source string comments, unlike translation comments, are associated with the string: it’s possible to have a comment in this section even if the string doesn’t have any suggestion or translation yet.

They’re designed for team members to have a conversation about the source string, for example to clarify its meaning, or to get more information from project managers.

### Machinery

> Machinery available from the translation view shows possible translations from a variety of sources. These sources include:

Reviewer: ```suggestion
Machinery shows possible translations from a variety of sources. These sources include:
```

* [Google Translate](https://translate.google.com).
* [SYSTRAN](https://www.systran.net/).
* [Caighdean](https://github.com/kscanne/caighdean).
* [Bing Translator](https://www.bing.com/translator) (not currently enabled on pontoon.mozilla.org).

In addition, the user has the ability to search for translations containing words via [`Concordance search`](#concordance-search).

In the tab, the number of entries is visible alongside the `MACHINERY` title in white. If any of the machinery matches are from translation memory, the number of matches will appear separately in green. For example, the screenshot below shows `2+1`, where the green `2` represents the two matches from translation memory and the `1` represents a machinery entry from Google Translate.

![Machinery tab](../../assets/images/pontoon/ui/machinery.png)

At the top of each entry, a diff view compares the current source string and the source string from the machinery entry. Strikethrough text highlighted in red indicates text that is present in the source string but not in the machinery source string, while text highlighted in green indicates text available only in the machinery source string.

> To the right of the entry, the number in green shows the percent match between the machinery string and the source string. The higher the percentage, the more likely the machinery is to be useful and a 100% match indicates that the source and machinery string are the same. The machinery source is listed in gray. Upon clicking the gray text, the machinery source will be opened in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory source.

Reviewer: ```suggestion
To the right of the entry, the number in green shows the percent match between the machinery source string and the current source string. The higher the percentage, the more likely machinery is to be useful; a 100% match indicates that the sources for the current string and for the machinery string are identical.

The origin of the machinery entry is listed in gray above the source string. Clicking the gray text will open the origin in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory.
```

I just realized this paragraph doesn't make a lot of sense, also mixing `source` for different meaning.

Author: Much simpler, thanks for the suggestion.

![Concordance search](../../assets/images/pontoon/ui/concordance_search.png)

#### Stand-alone machinery

Pontoon machinery is also available as a [stand-alone web page](https://pontoon.mozilla.org/machinery/). Use the `Machinery` link in the main navigation to access the page (this is not accessible from the translation workspace).

To use the machinery, select the target locale from the dropdown menu to the right of the searchbar. Then input the English source string into the searchbar and press Enter. This will display all matching machinery entries. Information is organized exactly as in the translation workspace.

### Locales

The locales tab shows approved translations from Pontoon projects in other [locales](glossary.md#locale).

![Locales tab](../../assets/images/pontoon/ui/locales.png)

Next to the `LOCALES` title, the number of available entries is visible. The number for preferred locales is green, while the number for all other locales is in gray.

Users can select locales to appear at the top of their Locales tab as a preferred locale. To add a locale to the preferred locale list, access the [user settings](users.md#locale-settings) page.

> Entries in the `LOCALES` tab above the green line are preferred locales. Entries below the green line are not preferred locales.

Reviewer: ```suggestion
Entries in the `LOCALES` tab above the green line are preferred locales. Non preferred locales are displayed below the green line, sorted alphabetically by language name.
```

> At the left of the entry, the translation for the source string in the selected locale is visible. At the right of the entry, the language name is visible in gray, while the locale code is displayed in green.

Reviewer: ```suggestion
Each row displays the translation for the source string in the selected locale. Above each entry, the language name is visible in gray, while the locale code is displayed in green.
```

> The `LOCALES` tab is useful for seeing what general style choices are made by other localization communities. When encountering a difficult string, a translator can reference the methods that have been used by other languages in making a stylistic decision.

Reviewer: ```suggestion
The `LOCALES` tab is useful for seeing what general style choices are made by other localization communities. When encountering a difficult string, a translator can use other locales as a source of inspiration.
```

> Note that when using the `LOCALES` tab the translator should always opt for fluency in the working locale. Languages vary linguistically on many levels. The locales tab can be extremely useful, but should be used carefully, and rarely as the sole translation resource for translation.

Reviewer: ```suggestion
Note that, when using the `LOCALES` tab, the translator should always opt for fluency in their own locale. Languages vary linguistically on many levels. The locales tab can be extremely useful, but should be used carefully, and rarely as the sole translation resource for translation.
```
